src/lib/tooltip/tooltip.spec.ts

```diff
@@ -432,6 +432,15 @@ describe('MdTooltip', () => {
       flushMicrotasks();
       expect(tooltipDirective._tooltipInstance).toBeNull();
     }));
+
+    it('should have rendered the tooltip text on init', fakeAsync(() => {
+      dispatchFakeEvent(buttonElement, 'mouseenter');
+      fixture.detectChanges();
+      tick(0);
+
+      const tooltipElement = overlayContainerElement.querySelector('.mat-tooltip') as HTMLElement;
+      expect(tooltipElement.textContent).toContain('initial tooltip message');
+    }));
```

Comment:
Does there need to be a test component w/ `OnPush` that would have exhibited the original behavior?